tensorflow/python/kernel_tests/array_ops_test.py

```diff
@@ -648,6 +648,31 @@ def testEllipsis(self):
       with self.assertRaisesRegexp(ValueError, "Multiple ellipses"):
         _ = checker[..., :, ...].eval()
 
+  @test_util.run_deprecated_v1
+  def testFetchMulti(self):
+    with self.session(use_gpu=True):
+      raw = [[[1,2,3], [4, 5, 6]], [[7,8,9], [10, 11, 12]]]
+      checker = StridedSliceChecker(self, raw)
+
+      _ = checker[[0, 1, 1], [1, 0, 1], [1, 0, 2]]
+      _ = checker[[0, 1, 1], [1, 0, 1]]
+      _ = checker[[0, 1], [1, 0]]
+      _ = checker[[0], [1]]
+  
+  @test_util.run_in_graph_and_eager_modes
+  def testFetchMultiEager(self):
+    with context.eager_mode():
```

Comment:
If you enter eager mode you're not evaluating the test in graph mode

Reply:
Hmm, So I don't need the decorator `@test_util.run_in_graph_and_eager_modes` here, right?

Reply:
No, just remove the context.eager_mode line so this one test applies to both eager and graphs

Reply:
Hmm, a quick question here is if we remove the context.eager_mode and use `@test_util.run_in_graph_and_eager_modes`, when it run in graph mode, could we also guarantee `self.assertAllEqual(raw[[0, 1, 1], [1, 0, 1], [1, 0, 2]].numpy(), 
	        raw_arr[[0, 1, 1], [1, 0, 1], [1, 0, 2]])` is available?